bundle-workflow/python/test_bwc.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python
+
+import os
+import argparse
+import sys
+import subprocess
+
+from manifests.bundle_manifest import BundleManifest
+from git.git_repository import GitRepository
+from test_workflow.local_test_cluster import LocalTestCluster
+from test_workflow.integ_test_suite import IntegTestSuite
+from paths.script_finder import ScriptFinder
+from system.temporary_directory import TemporaryDirectory
+
+
+def parse_arguments():
+    parser = argparse.ArgumentParser(description = "Test an OpenSearch Bundle")
+    parser.add_argument('--manifest', type = argparse.FileType('r'), help="Manifest file.")
+    parser.add_argument('--keep', dest = 'keep', action='store_true', help = "Do not delete the working temporary directory.")
+    args = parser.parse_args()
+    return args
+
+#TODO read from configuration file
+def is_component_test_supported(component):
```

Comment:
Move to be a property of `component`.

Reply:
Eventually we would not have a check and all components will be tested. 
i.e if any component test fails or tests do not exist we will flag them.